template-processors/base/Dockerfile

```diff
@@ -1,17 +1,18 @@
-FROM fedora
+FROM alpine:3.10
 
 ENV USER_UID=1001 \
     USER_NAME=gitopsjob \
     kubectl=kubectl
- 
-COPY kubernetes.repo /etc/yum.repos.d/kubernetes.repo 
-
-RUN dnf install -y git kubectl gettext python-pip jq findutils && \
-    pip install yq
 
 COPY bin /usr/local/bin
-RUN /usr/local/bin/user_setup
+
+RUN \
+    apk add --no-cache bash curl ca-certificates git gettext jq findutils py-pip && \
+    curl -L https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl -o /usr/bin/kubectl && \
```

Comment:
I would prefer to pin to a specific version of `kubectl` instead of always using `https://storage.googleapis.com/kubernetes-release/release/stable.txt`.